src/pint/models/parameter.py

```diff
@@ -507,6 +507,8 @@ def as_parfile_line(self, format="pint"):
             log.warning(
                 f"Changing KOM from DT92 convention to IAU: this will not be readable by PINT"
             )
+        elif self.name == "DMDATA" and not format.lower() == "pint":
```

Comment:
Don't we have an `intParameter` class? If so the right way to make this change would be to make `DMDATA` an `intParameter`. Or perhaps a `boolParameter`, since it is, and those accept `0`/`1` as input.

Reply:
I'm not sure.  `DMDATA` already is a `boolParameter`. The question isn't whether or not it can be read in correctly but how to change the output depending on desired format.  If all `boolParameters` want to be `T/F` for `PINT` and `1/0` for `TEMPO(2)` then we can do a more general change (and maybe that's the case - I haven't dug into it).  But this just fixes the specific output case that was causing a problem downstream.

Reply:
So parsing the input (0.0 vs. 0 vs. F) should still be the same.

Reply:
Is there any reason not to have DMDATA, at least, always output 0/1? PINT reads those happily. Perhaps when a boolParameter is created one could specify whether it is written as Y/N or 0/1? So where DMDATA is created we'd specify that it was just always written as 0/1?